numba_dppy/dpctl_iface/usm_shared_allocator_ext.c

```diff
@@ -24,6 +24,7 @@
 
 #include "numba/_pymodule.h"
```

Comment:
The file no longer is just `usm_shared`. We can rename it to `usm_allocators_ext`.